dune.module

```diff
@@ -4,7 +4,7 @@
 
 #Name of the module
 Module: dumux-precice
-Version: develop
+Version: 1.0.0
 Maintainer: alexander.jaust@ipvs.uni-stuttgart.de
```

Comment:
Do want to change the email address/the maintainer here?

Reply:
Yes, I have changed it to my email.

Reply:
Great. :+1: